curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646ae477486c4d02bf4e78f3.md

```diff
@@ -0,0 +1,65 @@
+---
+id: 646ae477486c4d02bf4e78f3
+title: Step 9
+challengeType: 0
+dashedName: step-9
+---
+
+# --description--
+
+A set of theme colors will be provided for you to use in the `:root` CSS selector. You will target the `body` element in your CSS and: 
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed